src/include/homestore/replication/repl_dev.h

```diff
@@ -270,6 +270,12 @@ class ReplDevListener {
     virtual void on_commit(int64_t lsn, sisl::blob const& header, sisl::blob const& key,
                            std::vector< MultiBlkId > const& blkids, cintrusive< repl_req_ctx >& ctx) = 0;
 
+    /// @brief periodically called to notify the lastest committed lsn to the listener.
+    ///
+    /// @param lsn - The log sequence number
```

Comment:
Concerning any heavy/blocking operations in the CB as it will block the DC_LSN flushing.

Lets start with putting clear comment here

Reply:
comments updated, ptal